django/views/i18n.py

```diff
@@ -218,51 +215,15 @@ def get_javascript_catalog(locale, domain, packages):
         paths.append(path)
     # add the filesystem paths listed in the LOCALE_PATHS setting
     paths.extend(reversed(settings.LOCALE_PATHS))
```

Comment:
Sorry it is me anoying again... The code from trans_real.py already calls `_add_local_translations` which does exactly that, load locales from the `settings.LOCALE_PATHS` folder. So is this line actually redundant? Or should trans_real only call the `_add_local_translations` if domain != 'django' ?

I'm also not sure of why for the jsi18n view it loads languages in such a different way (no caching, and packages are required where by default only the django.conf is loaded). Is it because building a complete jsi18n would be too big or heavy and rarely required? Remember that any production deploy will usually have this file cached so building it shouldn't be too expensive and right now loading the complete Translation object over and over might be as expensive or worse. This last thing might be a topic for another ticket but I thought it was worth mentioning since this is being worked on.